src/__tests__/test.js

```diff
@@ -1,23 +1,26 @@
 import Stream from 'stream';
 import {expect} from 'chai';
-import ava from 'ava';
+import test from 'ava';
 import Vinyl from 'vinyl';
-import svgmin from '..';
+import svgmin from '../index.js';
 
 const head = '<?xml version="1.0" encoding="utf-8"?>';
-const doctype = '<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">';
-const fullsvg = '<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve">' +
-            '<circle cx="50" cy="50" r="40" fill="yellow" />' +
-            '<!-- test comment --></svg>';
+const doctype =
+    '<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">';
```

Comment:
Maybe increase the printwidth (via prettierrc) so that the diff doesnt have so much linewraps?

Reply:
Possibly better might be to move these fixtures out into svg files and read them in from the test instead?

Reply:
Sounds good.

Reply:
@rejas Done, note that whitespace is controlled by `xo --fix` for now rather than `prettier --write`. I'd like at some point to get this all automated if possible. 👍